compiler/src/dmd/globals.d

```diff
@@ -82,6 +82,16 @@ enum CLIIdentifierTable : ubyte
     All      = 4, /// The least restrictive set of all other tables
 }
 
+/// When a non-normalized identifier is found, what should happen?
+enum CLIIdentifierNormalization : ubyte
+{
+    default_       = 0, /// The default normalization strategy
+    SilentlyAccept = 1, /// Silently accept the bad string
+    Deprecate      = 2, /// Emit a deprecation but allow
+    Warning        = 3, /// Emit a warning but allow
+    Error          = 4  /// Emit an error and fail
+}
+
```

Comment:
What is Output?

Reply:
The struct that followed this comment?

No idea, I didn't introduce that.

Reply:
ok